spec/migrations/add_last_active_role_to_people_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, Hitobito AG. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+require 'spec_helper'
+migration_file_name = Dir[Rails.root.join('db/migrate/20230615084630_add_last_active_role_to_people.rb')].first
+require migration_file_name
+
+
+describe AddLastActiveRoleToPeople do
+
+  before(:all) { self.use_transactional_tests = false }
+  after(:all)  { self.use_transactional_tests = true }
+
+  let(:migration) { described_class.new.tap { |m| m.verbose = false } }
+
+  after do
+    Person.delete_all
+  end
+
+  context '#up' do
+    let!(:people_without_active_roles) do
+      10.times.collect do
```

Comment:
Here we are testing the same thing with 10 people (which does not add value imho but slows down tests)